ci/submit_cython_cache.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+CACHE_File="$HOME/.cache/cython_files.tar"
+rm -rf $CACHE_File
+home_dir=$(pwd)
+ls "$TRAVIS_BUILD_DIR/pandas/"
+tar cf "$CACHE_File" \
+"$TRAVIS_BUILD_DIR"/pandas/{index,algos,lib,tslib,parser,hashtable}.c \
```

Comment:
I have added saslib but the follow are not being cythoned at the moment so are not creating a compiled file:

```
pandas/src/generated.pyx
pandas/src/inference.pyx
pandas/src/join.pyx
pandas/src/offsets.pyx
pandas/src/properties.pyx
pandas/src/reduce.pyx
pandas/src/skiplist.pyx
```